fragments/rbd/test_rbd_script_override.yaml

```diff
@@ -0,0 +1,12 @@
+# The purpose of this file implements the requirements of OCS integration.
```

Comment:
Could we rename this fragment to `upstream-workunits`

Reply:
sure pragadeesh